storybook/preview.ts

```diff
@@ -2,11 +2,10 @@ import { Preview } from '@storybook/react';
 import './global.css';
 import { DocsPage, DocsContainer } from '@storybook/addon-docs';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { withPerformance } from 'storybook-addon-performance';
+// import { withPerformance } from 'storybook-addon-performance';
```

Comment:
Doesn't work with React19?

Reply:
Based off of @ckifer branch. I don't know why he removed that. Though the commit was marked as "revert later" so I don't think that's relevant. I think the idea was just to force React 19, check that unit tests pass, revert React 19 and merge.

Reply:
Exactly. I had to upgrade master to storybook 8 to get things to behave. Idk why this works in our 3.x branch but not here, maybe I just didn't try hard enough. But either way it doesn't matter it was just to get R19 force installed so I could run the tests